lib/marc/controlfield.rb

```diff
@@ -40,31 +40,26 @@ def initialize(tag,value='')
     # Two control fields are equal if their tags and values are equal.
 
     def ==(other)
-      if @tag != other.tag
-        return false 
-      elsif @value != other.value
-        return false
-      end
-      return true
+      tag == other.tag && value == other.value
     end
 
     # turning it into a marc-hash element
     def to_marchash
       return [@tag, @value]
```

Comment:
Personally, I disagree with the style advice to avoid returns except to terminate early, I think they can frequently make the code more clear in other cases, this one included. but I don't care strongly.